src/components/EventModal/EventOwner.tsx

```diff
@@ -157,7 +157,7 @@ export default function EventOwner(props: EventOwnerProps) {
             return filtered as string[];
           }}
           renderInput={(params) => {
-            return <TextField {...params} variant="outlined" label="Add user(s) with email" />;
+            return <TextField {...params} variant="filled" label="Add user(s) with email" />;
```

Comment:
For some reason, outlined isn't showing up. Maybe it has to do with the params passed in but I've changed it to filled for now.

Reply:
This is already fixed on master

Reply:
Ok, reverted change